app/src/androidTest/java/com/github/se/cyrcle/model/parking/ParkingRepositoryFirestoreTest.kt

```diff
@@ -26,13 +23,6 @@ class ParkingRepositoryFirestoreTest {
     FirebaseApp.initializeApp(context)
 
     val db = FirebaseFirestore.getInstance()
-    val settings = firestoreSettings {
```

Comment:
These lines were causing issues with an error like: `Firebase error: cannot set settings more than once`.
I guess the underlying reason is that `MainActivityTest.kt` runs first and since it initializes our app, it sets the cache settings for the whole test run.